repair/row_level.cc

```diff
@@ -892,9 +892,9 @@ class repair_meta {
     future<repair_hash_set>
     working_row_hashes() {
         return do_with(repair_hash_set(), [this] (repair_hash_set& hashes) {
-            return do_for_each(_working_row_buf, [&hashes] (repair_row& r) {
+            return do_for_each(_working_row_buf, [&hashes] (repair_row& r) mutable {
```

Comment:
I don't think this mutable is strictly needed.

Reply:
I think so too. But it seems the compiler is happier this way. It won't hurt to add the mutable.